### PR TITLE
NFDIV-1481: Bring dev env inline with new access management changes

### PR DESCRIPTION
**Pull request by tomxelliott**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/NFDIV-1481

### CODE REVIEW

File: bin/am-add-role-assignment.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+classification=${4}
+grantType=${5}
+roleCategory=${6}
+readOnly=${7}
+startTime="now()"
+endTime="now() + interval '10 years'"
+attributes=${8}
+authorisations=${9}
+
+psql -h localhost -p ${DB_EXTERNAL_PORT} -d role_assignment -U nfdiv -c "INSERT INTO role_assignment (id, actor_id_type, actor_id, role_type, role_name, classification, grant_type, role_category, read_only, begin_time, end_time, attributes, created, authorisations) VALUES ('$(uuidgen)', 'IDAM', '${userId}', '${type}', '${roleName}', '${classification}', '${grantType}', '${roleCategory}', ${readOnly}, ${startTime}, ${endTime}, '${attributes}', 'now()', ${authorisations})" -q
```
Comment (linusnorton) [inline bin/am-add-role-assignment.sh]: we should just default this

Comment (linusnorton) [reply bin/am-add-role-assignment.sh]: like this `${DB_EXTERNAL_PORT:-3301}`

Comment (linusnorton) [reply bin/am-add-role-assignment.sh]: 5432 is the port number :)

File: docker-compose.yml [hunk 2]
```diff
@@ -457,6 +457,43 @@ services:
...
+      ROLE_ASSIGNMENT_IDAM_ADMIN_PASSWORD: "${CCD_SYSTEM_UPDATE_PASSWORD}"
+      ROLE_ASSIGNMENT_IDAM_ADMIN_SCOPE: profile openid roles search-user
+      CCD_DATA_STORE_URL: http://ccd-data-store-api:4452
+      AZURE_APPLICATIONINSIGHTS_INSTRUMENTATIONKEY:
+      REFORM_SERVICE_NAME: am-role-assignment-service
+      REFORM_TEAM: am
+      REFORM_ENVIRONMENT: local
+    ports:
+      - 4096:4096
+    depends_on:
```
Comment (linusnorton) [inline docker-compose.yml:491]: remove these, I don't think they do what is required.

File: bin/am-add-role-assignment.sh (hunk 1 above)
Comment (tomxelliott) [reply bin/am-add-role-assignment.sh]: Will default 👍

Comment (linusnorton) [reply bin/am-add-role-assignment.sh]: actually, when is this script called?

**Review by mark-butler-solirius — APPROVED** [review]

**Pull request merged**